UI/Browser.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#ifndef BROWSER_H
+#define BROWSER_H
+
+#include "imgui.h"
```

Comment:
Libraries not written by us should be formatted with angle brackets: `<imgui.h>`

Reply:
Fixed